tests/lax_numpy_test.py

```diff
@@ -2252,7 +2256,7 @@ def onp_fun(*args):
            "_a_shape={}_axis={}_keepdims={}".format(
              jtu.format_shape_dtype_string(a_shape, a_dtype),
              axis, keepdims),
-         "a_rng": jtu.rand_default(), 
+         "a_rng": jtu.rand_default(),
```

Comment:
It occurs to me that this is a bug too: we shouldn't instantiate rng factories in the test harness.